ChatQnA/tests/test_compose_on_rocm_vllm.sh

```diff
@@ -0,0 +1,265 @@
+#!/bin/bash
```

Comment:
`ChatQnA/tests/test_compose_on_rocm_vllm.sh` --> `ChatQnA/tests/test_compose_vllm_on_rocm.sh`   
The naming format will impact CI trigger.

Reply:
For this file name, `rocm_vllm` will be recognized as a hardware label, but we just need `rocm`. So it need to be update like what I commented.

Reply:
Accepted. Let's put everything in one place for the Arm architecture.